Samtools/1.10/View.nf

```diff
@@ -0,0 +1,18 @@
+process View {
+    tag {"Samtools View ${sample_id}"}
+    label 'Samtools_1_10'
+    label 'Samtools_1_10_View'
+    container = 'quay.io/biocontainers/samtools:1.10--h9402c20_2'
+    shell = ['/bin/bash', '-euo', 'pipefail']
+
+    input:
+    tuple sample_id, file(bam_file), file(bai_file)
+
+    output:
+    tuple sample_id, file("${bam_file.baseName}.sam")
+
+    script:
+    """
+    samtools view ${params.optional} ${bam_file} ${params.region} > ${bam_file.baseName}.sam
```

Comment:
Does this also work without region? If not, include region as optional parameter.

Reply:
I followed the tool regarding parameter names, see: http://www.htslib.org/doc/samtools-view.html

Region is indeed optional and you are still able to do this:
```
include View from 'View.nf' params(optional:'', region:'')
```

You can't specifiy a region using `params.optional`.